app/src/main/res/layout/language_item_new.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/background_elevation_1"
+    android:gravity="center_vertical"
+    android:paddingStart="@dimen/base_margin_triple"
+    android:paddingTop="@dimen/market_item_padding"
+    android:paddingEnd="@dimen/base_margin_triple"
+    android:paddingBottom="@dimen/market_item_padding">
+
+    <TextView
+        android:id="@+id/language"
+        style="@style/Body.Regular"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        tools:text="Sverige" />
+
+    <RadioButton
+        android:id="@+id/radioButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="@drawable/ic_radio_button_unchecked"
+        android:button="@color/transparent"
+        android:clickable="false" />
```

Comment:
Is this correct?

Reply:
if you mean `android:clickable="false"` then yes, if it is true it creates some weird behaviour if you click on the radiobutton, since i want the user to be able to click on the whole view